.github/workflows/lint-oas.yml

```diff
@@ -0,0 +1,26 @@
+name: lint-oas
+
+on:
+  push:
+    paths:
+      - specificatie/*.yaml
+      - .github/workflows/lint-oas.yml
+    branches:
+      - '**'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - name: Install dependencies
+        run: npm install
+      - name: lint OAS
+        run: npm run oas:lint
+      - name: lint resolved OAS
```

Comment:
Hier ontbreekt de resolve OAS stap. Nu lint je alleen de laatst gecommitte genereervariant

Reply:
Misschien begrijp ik je niet goed maar dat is bij de BAG repo toch ook niet de bedoeling. Daar wil je toch ook geen genereervariant genereren die wordt immers gemaakt door het Kadaster. Lijkt mij dat daar alleen de variant die zij gemaakt hebben gelint moet worden.

Reply:
Als je niet zelf resolve, dan weet je niet of de ./specifcatie/openapi.yaml bestand kan worden geresolvet. Wil je dat toch niet, dan is het ook niet nodig om in deze action de geresolved variant te linten, want die is nl. van de vorige commit

Reply:
Ik heb de 2 ontbrekende regels toegevoegd.